src/scripts/farming/BerryDeal.ts

```diff
@@ -146,6 +146,18 @@ class BerryDeal {
             ItemList['Masterball'],
             1
         ));
+        list.push(new BerryDeal(
+            [
+                this.randomBerry(fourthGen),
+                this.randomBerry(fifthGen),
+            ],
+            [
+                SeededRand.intBetween(20, 40),
+                SeededRand.intBetween(5, 20),
+            ],
```

Comment:
I feel like this might be a little bit too cheap 🤷

Reply:
Feel free to nerf the price if you want. I just chose randomly